_inc/client/components/upgrade-notice-content/index.jsx

```diff
@@ -37,67 +170,69 @@ const UpgradeNoticeContent = moduleSettingsForm(
 
 					<div className="jp-upgrade-notice__enable-module">
 
-						<SettingsGroup
-							hasChild
-							disableInDevMode
-							module={ lazyImages }>
-
-							<ModuleToggle
-								slug="lazy-images"
-								disabled={ false }
-								activated={ this.props.getOptionValue( 'lazy-images' ) }
-								toggling={ this.props.isSavingAnyOption( 'lazy-images' ) }
-								toggleModule={ this.toggleModule }
+						<SettingsGroup hasChild>
+							<CompactFormToggle
+								checked={ siteAcceleratorStatus }
+								toggling={ togglingSiteAccelerator }
+								onChange={ this.handleSiteAcceleratorChange }
+								disabled={ ! canDisplaySiteAcceleratorSettings }
 							>
 								<span className="jp-form-toggle-explanation">
-									{ decodeEntities( lazyImages.description ) }
+									{ __( 'Enable site accelerator' ) }
 								</span>
-							</ModuleToggle>
+							</CompactFormToggle>
+							<FormFieldset>
+								<ModuleToggle
+									slug="photon"
+									disabled={ this.props.isUnavailableInDevMode( 'photon' ) }
+									activated={ this.props.getOptionValue( 'photon' ) }
+									toggling={ this.props.isSavingAnyOption( 'photon' ) }
+									toggleModule={ this.toggleModule }
+								>
+									<span className="jp-form-toggle-explanation">
+										{ __( 'Speed up image load times' ) }
+									</span>
+								</ModuleToggle>
+								<ModuleToggle
+									slug="photon-cdn"
+									disabled={ false }
+									activated={ this.props.getOptionValue( 'photon-cdn' ) }
+									toggling={ this.props.isSavingAnyOption( 'photon-cdn' ) }
+									toggleModule={ this.toggleModule }
+								>
+									<span className="jp-form-toggle-explanation">
+										{ __( 'Speed up static file load times' ) }
+									</span>
+								</ModuleToggle>
+							</FormFieldset>
 						</SettingsGroup>
 					</div>
 
-					<p>
-						{ __( 'We have also upgraded all our Premium plan customers to unlimited high-speed video storage ' +
-							'(up from 13GB), and significantly reduced the CSS and JavaScript assets that Jetpack downloads ' +
-							'when using features like infinite scroll and embedding rich content.' ) }
-					</p>
-
-					<h2>
-						{ __( 'Faster, more relevant search results' ) }
-					</h2>
-
-					<a href="https://wp.me/p1moTy-731" rel="noopener noreferrer" target="_blank">
-						<img src="https://jetpackme.files.wordpress.com/2018/02/jetpack-elasticsearch-powered-search.png" width="700" alt={ __( 'Elasticsearch' ) } />
-					</a>
-
-					<p>
-						{ __( 'Our faster site search is now available to all Professional' +
-							' plan customers. This replaces the default WordPress search with an Elasticsearch-powered infrastructure that returns faster, more ' +
-							'relevant results to users.' ) }
-					</p>
-
 					<div className="jp-dialogue__cta-container">
 						<Button
 							primary={ true }
-							href="https://jetpack.com/?p=27095"
+							href="https://jetpack.com/support/site-accelerator/"
```

Comment:
Do we want this to open in the same tab/window or spin up a new one?

The link currently 404s, so just need to be sure it actually goes lives before the code ships.

Reply:
> Do we want this to open in the same tab/window or spin up a new one?

That's a good question. I don't know. @joanrho What do you think?

> need to be sure it actually goes lives before the code ships.

Yup, it's on Quill's list to review and publish at launch (cc @macmanx2).

Reply:
Confirming that, when published, the link will be `https://jetpack.com/support/site-accelerator/`